PalasoUIWindowsForms/ImageToolbox/AcquireImageControl.cs

```diff
@@ -46,36 +47,10 @@ void _galleryControl_ImageChanged(object sender, EventArgs e)
 		private void OnGetFromFileSystemClick(object sender, EventArgs e)
 		{
 			SetMode(Modes.SingleImage);
-//#if MONO
-//			            using (var dlg = new OpenFileDialog())
-//            {
-//                if (string.IsNullOrEmpty(ImageToolboxSettings.Default.LastImageFolder))
-//                {
-//                    dlg.InitialDirectory = System.Environment.GetFolderPath(Environment.SpecialFolder.MyPictures);
-//                }
-//                else
-//                {
-//                    dlg.InitialDirectory = ImageToolboxSettings.Default.LastImageFolder;
-//                }
-//
-//                dlg.Filter = "picture files|*.png;*.tif;*.tiff;*.jpg;*.jpeg;*.bmp;";
-//                dlg.Multiselect = false;
-//                dlg.AutoUpgradeEnabled = true;
-//				if (DialogResult.OK == dlg.ShowDialog())
-//                {
-//                    _currentImage = PalasoImage.FromFile(dlg.FileName);
-//                    _pictureBox.Image = _currentImage.Image;
-//                    ImageToolboxSettings.Default.LastImageFolder = Path.GetDirectoryName(dlg.FileName);
-//                    ImageToolboxSettings.Default.Save();
-//                    if (ImageChanged != null)
-//                        ImageChanged.Invoke(this, null);
-//                }
-//			}
-//#else
 #if MONO
 			using (var dlg = new OpenFileDialog())
 #else
-			//The primary thing this OpenFileDialogEx buys us is that with the standard one, there's
+			//The primary thing this OpenFileDialogWithViews buys us is that with the standard one, there's
 			//no way pre-set, what "view" the user gets. With the standard dialog,
 			//we had complaints that a user had to change the view to show icons *each time* they used this.
 			//Now, OpenFileDialogWithViews still doesn't let us read (and thus remember) the selected view.
```

Comment:
this comment could be further improved.  how about

    //The primary thing this OpenFileDialogWithViews buys us is the ability to default to large icons.